### PR TITLE
Adds Atmos Devices to Meta Chapel

### LINKED ISSUE #10227: [Bug]: Meta Chapel has Zero Atmospheric Devices
### BYOND Version

_No response_

### Round Date

2023/11/25

### Round ID

46740

### Test merges

_No response_

### Describe the bug

This image will explain it all...
image

No vents, no scrubbers...

### What did you expect?

I expected Atmos in my Atmos simulator.

### Reproducible steps

_No response_

### PR DESCRIPTION
**Pull request by Programs-The-Station**
## About The Pull Request

Adds Vents and Scrubbers to Meta Chapel. This closes #10227 .

## Why It's Good For The Game

Atmos devices in my atmos simulator? Shocking!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

**Tiles removed to show piping**

image

image

image

image

image

</details>

## Changelog
:cl:
fix: NT Atmospherics Division have corrected the lack of atmospheric devices within the MetaStation Chapel.
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
> ## About The Pull Request
> Adds Vents and Scrubbers to Meta Chapel

add 'closes #prnumber" so it closes the issue you made

**Comment by Programs-The-Station** [comment]
Yea, realized I forgot that right after hitting the submit button.

Oddly, I also didn't touch any area markers or APCs, yet the Map Merge tools added some of those changes.

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -47353,7 +47438,7 @@
 	pixel_y = 30
 	},
 /turf/open/floor/plasteel/grimy,
-/area/chapel/office)
+/area)
```
Comment (HowToLoLu) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Strange for this to be like this... did you touch the areas

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -53035,7 +53146,7 @@
 	dir = 4
 	},
 /turf/open/floor/plasteel,
-/area/maintenance/aft)
+/area)
```
Comment (HowToLoLu) [inline _maps/map_files/MetaStation/MetaStation.dmm]: Same here

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 3]
```diff
@@ -65134,7 +65295,7 @@
 	dir = 1
 	},
 /turf/open/floor/plating,
-/area/maintenance/aft)
+/area)
```
Comment (HowToLoLu) [inline _maps/map_files/MetaStation/MetaStation.dmm]: And here...

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 4]
```diff
@@ -66904,20 +67068,20 @@
...
 	},
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer4{
+	dir = 1
+	},
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer2{
+	dir = 1
+	},
 /turf/open/floor/plating,
-/area/maintenance/aft)
+/area)
```
Comment (HowToLoLu) [inline _maps/map_files/MetaStation/MetaStation.dmm:67084]: Here too....

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 5]
```diff
@@ -20799,8 +20799,14 @@
...
 /obj/structure/cable/yellow,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 5
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 5
+	},
 /turf/open/floor/plasteel/grimy,
-/area/chapel/office)
+/area)
```
Comment (HowToLoLu) [inline _maps/map_files/MetaStation/MetaStation.dmm:20809]: And here.

**Review by HowToLoLu — COMMENTED** [review]
The looks like the apcs and stuff *have* been touched somehow, and there's a bunch of typeless areas now. I'd double check the map file.

**Review by PestoVerde322 — CHANGES_REQUESTED** [review]
Yep, something defineley went wrong with the zoning here:

immagine

**Comment by Programs-The-Station** [comment]
Fixed the areas. First mapping PR, so figuring out the toolchain. I think I did accidentally delete the objects, probably when I was using my track pad.

**Review by PestoVerde322 — APPROVED** [review]
Now it's good ot go, approved.

**Pull request merged**